src/ThgHostingClient.php

```diff
@@ -1090,4 +1090,125 @@ public function getSshKeyById(int $sshId): array
     {
         return $this->request(ThgHostingClient::GET, "/ssh-keys/$sshId");
     }
+
+
+    /**
+     * Get server inventory list
+     *
+     * @param int|null    $datacenterId
+     * @param string|null $cpuBrand
+     * @param int|null    $minCpuCores
+     * @param int|null    $maxCpuCores
+     * @param float|null  $minCpuSpeed
+     * @param float|null  $maxCpuSpeed
+     * @param int|null    $minRam
+     * @param int|null    $maxRam
+     * @param string|null $storageType
+     * @param int|null    $minStorage
+     * @param int|null    $maxStorage
+     * @param int|null    $minNic
+     * @param int|null    $maxNic
+     * @param float|null  $minPrice
+     * @param float|null  $maxPrice
+     * @param bool|null   $raidEnabled
+     * @param string|null $sortBy
+     * @param string|null $direction
+     * @return array
+     * @throws ClientException
+     */
+    public function getServerInventory(
+        ?int $datacenterId,
+        ?string $cpuBrand,
+        ?int $minCpuCores,
+        ?int $maxCpuCores,
+        ?float $minCpuSpeed,
+        ?float $maxCpuSpeed,
+        ?int $minRam,
+        ?int $maxRam,
+        ?string $storageType,
+        ?int $minStorage,
+        ?int $maxStorage,
+        ?int $minNic,
+        ?int $maxNic,
+        ?float $minPrice,
+        ?float $maxPrice,
+        ?bool $raidEnabled,
+        ?string $sortBy,
+        ?string $direction
```

Comment:
I think we should not define the parameter as required fields to avoid putting all as NULL value when using this function.
So we can  add the default value to `null`.
```suggestion
        ?int $datacenterId = null,
        ?string $cpuBrand = null,
        ?int $minCpuCores = null,
        ?int $maxCpuCores = null,
        ?float $minCpuSpeed = null,
        ?float $maxCpuSpeed = null,
        ?int $minRam = null,
        ?int $maxRam = null,
        ?string $storageType = null,
        ?int $minStorage = null,
        ?int $maxStorage = null,
        ?int $minNic = null,
        ?int $maxNic = null,
        ?float $minPrice = null,
        ?float $maxPrice = null,
        ?bool $raidEnabled = null,
        ?string $sortBy = null,
        ?string $direction = null
```